src/PointwiseFunctions/AnalyticSolutions/Xcts/Schwarzschild.cpp

```diff
@@ -0,0 +1,350 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticSolutions/Xcts/Schwarzschild.hpp"
+
+#include <array>
+#include <cmath>
+#include <cstddef>
+#include <ostream>
+#include <pup.h>
+#include <utility>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"               // IWYU pragma: keep
+#include "NumericalAlgorithms/RootFinding/NewtonRaphson.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+/// \cond
+namespace Xcts::Solutions {
+
+std::ostream& operator<<(std::ostream& os,
+                         const SchwarzschildCoordinates& coords) noexcept {
+  switch (coords) {
+    case SchwarzschildCoordinates::Isotropic:
+      return os << "Isotropic";
+    default:
+      ERROR("Unknown SchwarzschildCoordinates");
+  }
+}
+
+template <>
+double Schwarzschild<
+    SchwarzschildCoordinates::Isotropic>::radius_at_horizon() noexcept {
+  return 0.5;
+}
+
+// Conformal spatial metric
+
```

Comment:
I suggest removing the blank lines between the comment and the code -- unless if the idea is that you are about to add a bunch of specializations for different coordinates?

(here and below)

Reply:
that is indeed my plan :)